### DISCUSSION TITLE
There is a "marketplace" in Github, how can I use for it?

### DISCUSSION THREAD

**Discussion by massimo1220** [community/community#33403] [Apps, API and Webhooks] [answered]
As you know, there is a "marketplace" in Github, how can I use for it?

**Comment by crypto-demigod** [comment, marked as answer]
Hi, @DevStar0202 

The answer is:
Adding an action from GitHub Marketplace
Navigate to the action you want to use in your workflow.
Under "Installation", click to copy the workflow syntax.
Paste the syntax as a new step in your workflow. ...
If the action requires you to provide inputs, set them in your workflow.

I hope my answer helped you.

**Reply by massimo1220** [reply]
Oh, @superherodemigod . my friend.
Thanks for your kindly help.

Thanks.

**Reply by crypto-demigod** [reply]
It's my pleasure.
